draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -586,28 +588,51 @@ Providing the proxy with the final SvcDomainName has several benefits:
 
 # DNS Server Behavior {#server-behavior}
 
+## Authoritative servers
+
 When replying to an SVCB query, authoritative DNS servers SHOULD return
-A, AAAA, and SVCB records (as
-well as any relevant CNAME records) in the Additional Section for any
+A, AAAA, and SVCB records (as well as any relevant CNAME or
+{{!DNAME=RFC6672}} records) in the Additional Section for any
 in-bailiwick SvcDomainNames.
 
+## Recursive resolvers
+
 Recursive resolvers that are aware of SVCB SHOULD ensure that the client can
 execute the procedure in {{client-behavior}} without issuing a second
-round of queries, by following this procedure while constructing a response
-to a stub resolver for an SVCB record query:
+round of queries, by incorporating all the necessary information into a
+single response.  For the initial SVCB record query, this is just the normal
+response construction process (i.e. unknown RR type resolution under
+{{!RFC3597}}).  For followup resolutions performed during this procedure,
+we define incorporation as adding all Answer and Additional RRs to the
+Additional section, and all Authority RRs to the Authority section,
+without altering the response code.
```

Comment:
Perhaps add:

"Recursive resolvers MAY respond to the SVCB Answer with 
only a subset of Additional RRs when the response would
be larger than is permitted or when responding before fully chasing 
dependencies would improve performance.  Additional information from
higher priority ServiceForm records SHOULD be prioritized over lower priority
ServiceForm records."

Reply:
OK.  I added this to the Performance Optimizations section.